build.yaml

```diff
@@ -31,11 +26,7 @@ steps:
        valueFrom: default_ns.name
    public: false
    secrets:
-    # hail|test-batch
-    - user-jwt-vkqfw
-    - gsa-key-2x975
-    # pipeline|test-user
-    - pipeline-test-0-1--hail-is-service-account-key
```

Comment:
I think deleting the pipeline service account will fail until my PR goes in. Trying to debug that still.